xmodule/studio_editable.py

```diff
@@ -12,7 +16,6 @@ class StudioEditableBlock(XBlockMixin):
 
     This class is only intended to be used with an XBlock!
     """
-    has_author_view = True
```

Comment:
since now all blocks inherit studio editable in cms, this shouldn't be here since there will be blocks that don't have author_view. 

I did it this way to keep the currently existing pattern but I'd personally prefer to change the `has_author_view(block):` implementation bellow to check if the block implements the author_view method instead of relying in a flag that is brittle imo